os_migrate/plugins/module_utils/filesystem.py

```diff
@@ -38,7 +38,11 @@ def load_resources_file(file_path):
     with open(file_path, 'r', encoding='utf8') as f:
         file_struct = yaml.safe_load(f)
 
-    file_os_migrate_version = file_struct.get('os_migrate_version', None)
+    if file_struct is not None:
+        file_os_migrate_version = file_struct.get('os_migrate_version', None)
+    else:
+        raise exc.EmptyYAMLFileError(file_path)
```

Comment:
This code block can be simplified to:

```
    if file_struct is None:
        raise exc.EmptyYAMLFileError(file_path)

    file_os_migrate_version = file_struct.get('os_migrate_version', None)
```

Basically, we check for the exception condition first, if it's present then we raise the exception. Then we continue with the happy path as usual. Not a blocker but it's a customary way to deal with error conditions, to keep the control flow simple, so it may be worth an update perhaps...